style.css

```diff
@@ -0,0 +1,951 @@
+/* font-family: 'Gothic A1', sans-serif;
+font-family: 'Lato', sans-serif;
+font-family: 'Rubik Dirt', cursive; */
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  list-style: none;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+#upper__nav {
+  display: none;
+}
+
+#lower__nav {
+  width: 100%;
+  height: 63px;
+  background: #fff;
+  display: flex;
+}
+
+.nav__menu-container {
+  position: relative;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav-toggle {
+  position: absolute;
+  left: 23px;
+  font-size: 1.5rem;
+  color: #534e4e;
+  cursor: pointer;
+  background: none;
+  display: flex;
+}
+
+.closeMenu {
+  display: flex;
+  justify-content: flex-start;
+  padding-left: 24px;
+  width: 100%;
+  height: 48px;
+  background: none;
+  font-size: 24px;
+  color: #606060;
+  cursor: pointer;
+}
+
+.lower__nav-logo {
+  width: 131px;
+}
+
+.lower__nav-menu {
+  margin: 0;
+  display: flex;
+  padding: 20px 0;
+  flex-direction: column;
+  align-items: center;
+  background: #3e3e3e;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  height: 100vh;
+  left: 100%;
+}
+
+.lower__nav-elements {
+  font: 30px/82px 'Lato', sans-serif;
+  color: #fff;
+  margin: 0 10px;
+  width: max-content;
+}
+
+.lower__nav-menu-visible {
+  left: 0;
+}
+
+#hero {
+  padding: 109px 12px 71px 12px;
+  background-image: url(https://wallpaperaccess.com/full/1968246.jpg);
+  background-size: cover;
+  font-family: 'Lato', sans-serif;
+}
+
+.hero__container-title {
+  font-size: 35px;
+  background-clip: text;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-image: url(Assets/img-bg-h1.jpg);
+  background-attachment: fixed;
+}
+
+.hero__container-subtitle {
+  color: #fa9200;
+  font-weight: 400;
+}
+
+.paragraphs {
+  width: 100%;
+  height: 85px;
+  color: #515151;
+  font: 400 12px/17px 'Lato', sans-serif;
+  font-style: normal;
+  letter-spacing: -0.12px;
+  border: 4px solid #fff;
+  background: #f8f8f8;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 17.101px 14px 16.899px 14px;
+  margin: 30px 0 20px 0;
+}
+
+.date {
+  color: #595959;
+  font: 900 20px/25px 'Gothic A1', sans-serif;
+  font-style: normal;
+  letter-spacing: -1.6px;
+}
+
+.location {
+  display: flex;
+  height: 36.762px;
+  flex-direction: column;
+  justify-content: center;
+  color: #868686;
+  font: 700 13px/25px 'Gothic A1', sans-serif;
+  letter-spacing: -0.26px;
+}
+
+.main__program-container {
+  background: #868686;
```

Comment:
Kindly make use of the right colors as required in the design so as to make your project match the given design as this is a requirement for this milestone😉.

<img width="457" alt="image" src="https://github.com/sinruidotec/Capstone-Project/assets/64643995/7c31b5d1-082e-4734-baa8-2a0d41311ba4">